unison-src/transcripts/io.md

```diff
@@ -191,3 +191,20 @@ testSystemTime _ =
 .> add
 .> io.test testSystemTime
 ```
+
+### Get directory contents
+
+```unison
+testDirContents : '{io2.IO} [Result]
+testDirContents _ = 
+  test = 'let
+    tempDir = newTempDir "dircontents"
+    c = reraise (directoryContents.impl tempDir)
+    check "directory size should be"  (size c == 2)
```

Comment:
This testing is pretty thin - I'd create a file or two in the directory and check the name of the file coming back from getDirectory equals what was created. If you're just checking length then it's not testing that the Text values coming back from Haskell are encoded correctly as Unison text.

I did a double take that an empty directory has size 2 but I guess that's for current directory and parent? I feel like a version that filters these out would be more commonly used (think of like walking a directory tree) but that could also be done in pure Unison. Wdyt?